spec/system/admin/bulk_order_management_spec.rb

```diff
@@ -485,7 +485,8 @@
           displays_default_orders
         end
 
-        it "allows combining the order cycle and the pre-selected date with a custom date" do
+        xit "allows combining the order cycle and the pre-selected date with a custom date" do
+          # pending (#10192)
```

Comment:
To be pragmatic, maybe it's a good idea, since will be soon deleted via https://github.com/openfoodfoundation/openfoodnetwork/pull/10243

Reply:
Ok, thanks @jibees - I've moved this to a separate PR: #10286.